### ISSUE TITLE
whitespace issues

### ISSUE THREAD

**Issue by alvinwan** [alvinwan/TexSoup#2]
TexSoup objects, when printed, do not respect the "structure" (whitespace) of the original document. Ideally, this should be fixed before the 0.0.2 release.

**Comment by efege** [comment]
Hi, Alvin. I'm not sure if the problem illustrated by the following example is included in this issue #2.

```
#!/usr/bin/python3

from TexSoup import TexSoup

source = r"""
\begin{article}
\title {This title contains
a line break}
\end{article}
"""

soup = TexSoup(source)

for art in soup.find_all('article'):
    title = art.title.args[0]
    print(title)
```

The output is not what I expected, since the two lines in the title are joined with no whitespace at all:

```
This title containsa line break
```

BTW, thank you for the code!

**Comment by alvinwan** [comment]
@efege Whoa, thanks for letting me know - I will take a look
